### DISCUSSION TITLE
[BUG] Proof of Sync. ??

### DISCUSSION THREAD

**Discussion by adamsky-01** [Chia-Network/chia-blockchain#3265] [Support]
The last few days all my chia resources like time, hardware are wasted on syncing. My 8444 port is open (verified) but it seems like syncing will take forever. Even when connected to introducers syncing is extremally slow. 
Is there any official comment on this, like ...stop using version 1.1.2, it will never sync. or this version only works under linux, etc.?  

I was planning to do plotting but instead I'm trying to sync the last few days.

![image](https://user-images.githubusercontent.com/60489596/116252199-41a90c80-a767-11eb-8e57-6669fe14c309.png)

**Comment by sunyao920807** [comment]
You may be in multiple NAT, you can try to change to public IP address

**Comment by adamsky-01** [comment]
> You may be in multiple NAT, you can try to change to public IP address

It cannot be a case if all port testers reporting that the port 8444 is open on my IP, right?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
